test-blink.cpp

```diff
@@ -1,18 +1,21 @@
-// Dependency:
-// https://github.com/WiringPi/WiringPi
-//
 // Build:
-// g++ -Wall test-blink.cpp -lwiringPi -o test-blink
+// g++ -Wall test-blink.cpp -lgpiod -o test-blink
 
-#include <wiringPi.h>
+#include <gpiod.h>
 #include <cstdlib>
 #include <iostream>
 #include <cstring>
+#include <unistd.h>
 
-#define GPIO "GPIO"
+#define GPIO "line"
```

Comment:
Use SCREAMING_CASE for environment variables. 

Moreover, note that this needs to stay compatible with the other apps in the snap, as well as the hooks and documentation. So it is best to keep it as `GPIO`, short for GPIO Line.

Reply:
Okay done!